ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -208,10 +270,15 @@ open class SegmentedControl: UIControl {
     @objc open func insertSegment(withTitle title: String, at index: Int) {
         items.insert(title, at: index)
 
-        let button = createButton(withTitle: title)
+        let button = style == .tabs ? createTabButton(withTitle: title) : createSwitchButton(withTitle: title)
         buttons.insert(button, at: index)
         // TODO: Add option for animated addition?
-        addSubview(button)
+        if style == .tabs
```

Comment:
Can we improve this logic and use just the if statement and avoid 2 checks?
Something like:
```

        let button: UIButton
        if style == .tabs {
            button = createTabButton(withTitle: title)
            addSubview(button)
        } else {
            button = createSwitchButton(withTitle: title)
            containerView.addSubview(button)
        }

        buttons.insert(button, at: index)
```